ColorListGenerator/Color.swift

```diff
@@ -39,19 +39,21 @@ class Color {
         }
     }
 
-    class func colorFromHexString(value: String) -> NSColor? {
+    class func colorFromHexString(value: String, var alpha: CGFloat = 1.0) -> NSColor? {
```

Comment:
The argument `var alpha: CGFloat` is not used from any caller side, so the change seems not to be needed.